src/uct/ib/base/ib_device.h

```diff
@@ -99,13 +100,21 @@ typedef struct uct_ib_address {
 } UCS_S_PACKED uct_ib_address_t;
 
 
+/**
+ * PCI identifier of a device
+ */
+typedef struct {
+    uint16_t                    vendor;
+    uint16_t                    device;
+} uct_ib_pci_id_t;
```

Comment:
as I understand the feature we need to expose the subsystem_device and subsystem_vendor in addition to the vendor and device values.
the new subsystem_* appear in the PCI sysfs, but not in the verbs.

<vendor, device> = IC (ConnectX chipset)
<subsystem_vendor, subsystem_device> = Board/Card (EOM)

Reply:
for identifying HCA features (e.g DC or mlx5 support) we need chipset ID only